hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/FreezeUpgradeActions.java

```diff
@@ -39,21 +39,21 @@ public class FreezeUpgradeActions extends ReadableFreezeUpgradeActions {
     /**
      * Constructs a {@link FreezeUpgradeActions} instance
      *
-     * @param adminServiceConfig the network admin service configuration
+     * @param context the HandleContext
      * @param freezeStore the freeze store
      * @param executor the executor
      * @param upgradeFileStore the upgrade file store
      * @param nodeStore the node store
      * @param stakingInfoStore the staking info store
      */
     public FreezeUpgradeActions(
-            @NonNull final NetworkAdminConfig adminServiceConfig,
+            @NonNull final HandleContext context,
```

Comment:
Just Configuration is fine to be passed in

Reply:
Updated.